### DISCUSSION TITLE
Is autobahn-cpp still working in December 2024?

### DISCUSSION THREAD

**Discussion by malibu1966** [crossbario/autobahn-cpp#243] [Q&A]
Hi there, I was planning on making a win32 mingw/C++/Qt6/QML/WAMP client but I am unclear on how to build this and wonder if it is still possible.  I attempted to build the boost library but that instructed me to build a b2 binary first and the link to the b2 binary build gives me a 404.

Please help me not waste my time, does anyone know if the dependencies can be built today?  Otherwise I will switch to a Python/Pyside6/QML based client.

**Comment by oberstet** [comment]
Hi!

> win32 mingw

building boost for a semi-*NIX like mingw has always been a challenge (in my experience), more or less

personally, I would recommend:

`format C:` and move to a real Unix/Linux, or maybe the Linux that Windows nowerdays ships by default with (forgot the name, some subsystem whatever)

OR

as you note, use Python/Pyside6/QML, which is a very capable and advanced environment in general - and it _should_ work more or less decently on windows as well

if you absolutely want to have win32 mingw/C++ and Boost and everything, this is for the hard core people, expect issues and pain. nothing new. windows. my 2cts.

hope that helps,

cheers,
/Tobias

**Reply by malibu1966** [reply]
Ok that's what it seemed like to me.. the issues and pain part.  I will go with Python then.  Unfortunately I cannot force my company to use linux workstations so whatever I need to do needs to work on windows.  And yes I prefer linux too.. That's what I do all my development on.  Thank you for the quick response, as always.
